frontend/public/index.html

```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/saildog-logo.png" />
```

Comment:
You should keep the name `favicon.ico` (and the logo variants). They are referenced in the manifest file which is important for turning your webapp into a PWA.

Reply:
I have now added code that was automatically generated. Hope that is better.

Reply:
Just a precaution to save you some headache if you decide to turn it into a PWA ;)